packages/thirdweb/src/react/core/utils/walletIcon.ts

```diff
@@ -56,6 +50,7 @@ export const socialIcons = {
   telegram: telegramIconUri,
 };
 
+// TODO: this should return actual <svg> elements so they can be themed
```

Comment:
cc @MananTank @greg - didnt do it in this PR, but we should switch to using <svg> elements for icons like email/phone/passkey/wallet so we can theme those.

Right now you can't change the color of those on web.

Reply:
We need a separation between the "Wallet Image/icon" vs normal icon in UI - the wallet image/icon is not supposed to be themable but a normal icon should be